addons/map_gestures/functions/fnc_getProximityPlayers.sqf

```diff
@@ -37,7 +37,7 @@ switch (_this) do {
         _proximityPlayers append allUnits;
     };
     case (1): { // Players in Group
-        _proximityPlayers append group ace_player;
+        _proximityPlayers append (units ace_player);
```

Comment:
```suggestion
        _proximityPlayers append units ace_player;
````

Reply:
Why? Parentheses help readability without a single downside. It is used widely in the harshest standards to assure correct execution of code in the entire industry. I am not a fan of full MISRA for this use case, but this is a perfect example where parentheses outright improve readability (and possibly allow easy changing of commands that might have higher priority instead).

Reply:
I like commy's suggestion here. A matter of taste I think.